bentoctl/utils/usage_stats.py

```diff
@@ -1,4 +1,9 @@
+import typing as t
+
+import attr
+
 # These are internal apis. We will need to make sure update these when BentoML changes.
+from bentoml._internal.utils.analytics.schemas import EventMeta
```

Comment:
cc @ssheng another point we need to consider when discussing about API exposure post 1.0